utilities/src/test/java/org/craftercms/commons/config/YamlConfigurationTest.java

```diff
@@ -34,6 +34,7 @@ public class YamlConfigurationTest {
     @Before
     public void setUp() throws Exception {
         yamlConfiguration = new YamlConfiguration();
+        yamlConfiguration.loaderOptions.setMaxAliasesForCollections(50);
```

Comment:
@jmendeza please note this.

CC @yacdaniel it seems that we may need a quick validation.